mne_bids/tests/test_mne_bids.py

```diff
@@ -229,7 +278,66 @@ def test_cnt():
 
     raw_to_bids(subject_id=subject_id, session_id=session_id, run=run,
                 task=task, acquisition=acq, raw_file=raw_fname,
-                output_path=output_path, overwrite=True, kind='eeg')
+                output_path=output_path, write_mode='append', kind='eeg')
 
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
+
+
+def test_privates():
```

Comment:
the name is a bit weird. Alternatively, `test_json_tsv` ? or `test_generate_sidecar` ?